flow360/component/simulation/cloud.py

```diff
@@ -183,6 +190,18 @@ def _run(
             f"params argument must be a SimulationParams object but is of type {type(params)}"
         )
 
+    ##-- Getting the project length unit from draft and store in the SimulationParams
+    _resp = RestApi(ProjectInterface.endpoint, id=source_asset.project_id).get()
+    last_opened_draft_id = _resp["lastOpenDraftId"]
+    assert last_opened_draft_id is not None
+    _draft_with_length_unit = Draft.from_cloud(last_opened_draft_id)
+    length_unit = _draft_with_length_unit.retrieve_simulation_dict()[
```

Comment:
this adds 3 more http requests. It will significantly slow down overall experience. We need to talk to backend team on how limit number of http requests

Reply:
right, it would be best to get the length unit from just Geometry get so that all three here is eliminated.